### DISCUSSION TITLE
Redirecting me

### DISCUSSION THREAD

**Discussion by FreshImmuc** [YOURLS/YOURLS#3014] [Q&A] [answered]
Okay so when I create a YOURLS link, everything is fine but when I open it, I just get redirected to the parent directory. Also idk if this is an error but If I go to the url (without the /admin or so), I just see the folder thing... Is it supposed to show the files or what?

![image](https://user-images.githubusercontent.com/65733898/128616591-5d5f5cb6-6803-4046-b10d-ee95ec0a0762.png)

**Comment by dgw** [comment, marked as answer]
Follow the appropriate server configuration guide from [our wiki](https://github.com/YOURLS/YOURLS/wiki#quick-start). This is clearly not set up correctly.

**Reply by ozh** [reply]
Also : http://yourls.org/#FAQ;noindex

**Reply by FreshImmuc** [reply]
Yes. thanks this helped me.
